analysis/src/CompletionDecorators.ml

```diff
@@ -30,8 +30,6 @@ It is also possible to map a ReScript record to a JavaScript array by passing in
       [
         {|The `@deprecated` decorator is used to add deprecation notes to types, values and submodules. The compiler and editor tooling will yield a warning whenever a deprecated entity is being used.
 
-Alternatively, use the `@@deprecated` decorator to add a deprecation warning to the file level.
```

Comment:
Should this really be removed?

Reply:
Add here https://github.com/rescript-lang/rescript-vscode/pull/799/commits/e3e0463259bc85ee4ac86ebf71ec3fc581ebda33